### ISSUE TITLE
Resolve CORS error in Flask 

### ISSUE THREAD

**Issue by linglp** [Sage-Bionetworks/schematic#822]
**Is your feature request related to a problem? Please describe.**
Resolve CORS error using flask

**Describe the solution you'd like**
Have Flask API communicate directly with the visualization front-end. 

**How important is this feature?** Select from the options below:
• 🌋 Important - it's a blocker and can't do work without it

**When will use cases depending on this become relevant?** Select from the options below:
• Short-term - 2-4 weeks

**Additional context**

**Comment by linglp** [comment]
I have resolved the CORS issue. Here are the notes: 
* Launch API by using branch `develop-schema-viz-tool-cors`. This branch was created by using Mialy's original branch `develop-schema-viz-tool`. I added additional packages and some code for handling CORS
* Use branch `develop-flask-api` in schema_visualization github repo

About API performance: 
* It took around 1-2 seconds for "select dataset type" to be populated
* After selection, it took another second for the visualization to show up

Here's the screen shot of the visualization generated by the API calls (which looks a bit different from the one [on Observable](https://observablehq.com/d/c3fd85acfb34db59))
<img width="783" alt="Screen Shot 2022-07-19 at 2 45 48 PM" src="https://user-images.githubusercontent.com/55448354/179828809-b4681953-7a55-48f0-92e4-bb4946417a97.png">
